src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricsExporter.cs

```diff
@@ -32,7 +32,7 @@ namespace OpenTelemetry.Exporter
     /// Exporter consuming <see cref="Metric"/> and exporting the data using
     /// the OpenTelemetry protocol (OTLP).
     /// </summary>
-    [AggregationTemporality(AggregationTemporality.Both, AggregationTemporality.Cumulative)]
+    [AggregationTemporality(AggregationTemporality.Cumulative | AggregationTemporality.Delta, AggregationTemporality.Cumulative)]
```

Comment:
Not related to this PR, but I'm not following why the OTLP exporter is marked to prefer `Cumulative`. Seems to me the OTLP exporter should effectively not have a preference.

For a given service, is it currently not possible for some metrics to have a cumulative aggregation and others to have a delta aggregation?

Reply:
Ok, sorry, I _think_ this is making sense to me now. I'm still trying to keep all these things in my head. I continue going back to a time when there was discussion about being able to define aggregation temporality at the level of an individual instrument or view. This is not the case.

Aggregation temporality is essentially defined at the level of a pipeline - so like `MetricReader` -> `MetricExporter`.

Reply:
Let me put some concrete examples to help us align the understanding:

Prometheus - only supports Cumulative (implied that Cumulative is also the preference):
* The user doesn't have to specify temporality during the SDK configuration, everything will be exported as cumulative. In case the measurement being reported is Delta, there will be Delta->Cumulative conversion.
* If the user specified Cumulative explicitly, it won't harm
* If the user specified Delta explicitly, they will get some error telling them that the setup is wrong.

OTLP - supports both Cumulative and Delta (and Cumulative is the preference):
* The user doesn't have to specify temporality during the SDK configuration, everything will be exported as Cumulative. In case the measurement being reported is Delta, there will be Delta->Cumulative conversion.
* If the user specified Cumulative explicitly, it won't harm
* If the user specified Delta explicitly, there are two cases:
  * If the value being reported is Delta (e.g. Counter), it'll be reported as Delta, no conversion required
  * If the value being reported is Cumulative (e.g. ObservableCounter), there will be Cumulative->Delta conversion.

Reply:
@alanwest FYI https://github.com/open-telemetry/opentelemetry-specification/issues/731

Reply:
Thank you for this context! Defaulting to cumulative had seemed somewhat arbitrary to me, but Prometheus + load-balanced collectors = a bad time if delta. Prometheus seems to make everything trickier 😄.